helix-common/src/main/java/org/apache/helix/SystemPropertyKeys.java

```diff
@@ -78,4 +78,7 @@
   // System Property Metadata Store Directory Server endpoint key
   public static final String MSDS_SERVER_ENDPOINT_KEY =
       MetadataStoreRoutingConstants.MSDS_SERVER_ENDPOINT_KEY;
+
+  // System property for request timeout
+  public static final String HTTP_REQUEST_TIMEOUT = "http.request.timeout";
```

Comment:
Is this strictly for Helix REST? In that case, should we try to put this in a PropertyKey constant file in the helix-rest module?

Reply:
Done. Added a file called RestSystemPropertyKeys and added this key to that file.